src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -751,11 +743,29 @@ private async Task ValidateSignUpPoliciesAsync(Guid ownerId)
                     await _collectionRepository.CreateAsync(defaultCollection);
                 }
 
-                // push
-                var deviceIds = await GetUserDeviceIdsAsync(orgUser.UserId.Value);
-                await _pushRegistrationService.AddUserRegistrationOrganizationAsync(deviceIds,
-                    organization.Id.ToString());
-                await _pushNotificationService.PushSyncOrgKeysAsync(ownerId);
+                OrganizationUser orgUser = null;
+                if (ownerId != default)
+                {
+                    orgUser = new OrganizationUser
+                    {
+                        OrganizationId = organization.Id,
+                        UserId = ownerId,
+                        Key = ownerKey,
+                        Type = OrganizationUserType.Owner,
+                        Status = OrganizationUserStatusType.Confirmed,
+                        AccessAll = true,
+                        CreationDate = organization.CreationDate,
+                        RevisionDate = organization.CreationDate
+                    };
+
+                    await _organizationUserRepository.CreateAsync(orgUser);
+                    
+                    // push
```

Comment:
I'm not sure what this comment means in this context other than that the push services are called below it. That said, I think you pulled this from another area as I think I've seen this comment before in the project. Maybe we should stop adding it or add some more information?

Reply:
You've probably seen it on line 754 since that's where it was moved from. But I agree bit of a unnecessary comment.